Gemfile.jruby-1.9.lock

```diff
@@ -12,8 +12,6 @@ PATH
       jrjackson (~> 0.4.0)
       jrmonitor (~> 0.4.2)
       jruby-openssl (= 0.9.16)
-      logstash-core-event-java (= 5.4.0)
-      logstash-core-queue-jruby (= 5.4.0)
```

Comment:
This RED changes was confusing me, I think the lock should have been regenerated after we merge the `logstash-core-event` removal, I've looked at the lock file in the 5.4.0 artifact and it doesn't content any trace of theses removed lines.